tests/cross/src/test/scala/tests/highlight/Scala3DocumentHighlightSuite.scala

```diff
@@ -107,4 +107,28 @@ class Scala3DocumentHighlightSuite extends BaseDocumentHighlightSuite {
        |""".stripMargin,
   )
 
+  check(
+    "given-synthetic1",
+    """|given (usi@@ng i: Int): Double = 4.0
+       |val a = given_Double""".stripMargin,
+  )
+
+  check(
+    "given-synthetic2",
+    """|given (using i: Int): Double = 4.0
+       |val a = <<given_Doub@@le>>""".stripMargin,
+  )
+
+  check(
+    "given-not-synthetic1",
+    """|given <<`giv@@en_D`>>: Double = 4.0
+       |val a = <<`given_D`>>""".stripMargin,
+  )
+
+  check(
+    "given-not-synthetic2",
+    """|given <<`giv@@en_D`>>:Double = 4.0
+       |val a = <<`given_D`>>""".stripMargin,
+  )
```

Comment:
the `@@` should be in `val a` ? It's in given in previous test

Reply:
Ach, good catch. Fixed